api/generate_models.ts

```diff
@@ -33,6 +28,12 @@ async function generateModelsFromApi(
     })
   );
 
+  const definitions = api.definitions;
+  if (!definitions) {
+    console.log("No definitions found");
```

Comment:
I would say `No definitions found, skipping generation of model code` or something like that

Reply:
ok